src/fontstash.h

```diff
@@ -886,14 +886,40 @@ static int fons__allocFont(FONScontext* stash)
 	return FONS_INVALID;
 }
 
+FILE* multibyte_open(const char*filename, const char*mode)
```

Comment:
Internal functions are static, and prefixed with `fons__`, so this would be `static FILE* fons_fopen()`.
Also space after *